### ISSUE TITLE
bug(macOS): the install button does not install the keyboard, but download and stay cold on it 

### ISSUE THREAD

**Issue by MakaraSok** [keymanapp/keyman#7798]
### Describe the bug

<img width="300" alt="image" src="https://user-images.githubusercontent.com/28331388/203473281-e2523a5f-71a2-4534-9d93-e4be0e7dd187.png">

On macOS, the green button `Install keyboard` is doing half of what it says. From the user point of view, I am expecting the keyboard to install when the button is pressed (probably after the keyboard package is successfully downloaded).

On Windows, it works as expected. The install window shows up right after the button is pressed.
<img width="300" alt="image" src="https://user-images.githubusercontent.com/28331388/203474202-794b64db-5a27-4396-a489-b967f33aa8ec.png">

### Reproduce the bug

1. Go to keyman.com
2. Search for a keyboard, say "Hebrew"
3. Click on [Galaxie Greek/Hebrew (Positional)](https://keyman.com/keyboards/galaxie_greek_hebrew_positional)
4. Then click on the green button "Install keyboard"
5. See error - the keyboard doesn't get installed after it finishes downloading its package 

### Expected behavior

The keyboard package should have been triggered and the install window should pop up for users to install the keyboard.

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [X] Other - give details at bottom of form

### Keyman version

16.0.109-beta

### Operating system

macOS ventura 13.0.1

### Device

MBP 2020

### Target application

Safari (keyman.com)

### Additional context

The install button doesn't install the keyboard it says it does on its label.

**Comment by mcdurdin** [comment]
We should probably change the text to "Download keyboard" and have instructions on the final step to click on the keyboard in the browser downloads to complete installation

**Comment by darcywong00** [comment]
> We should probably change the text to "Download keyboard" and have instructions on the final step to click on the keyboard in the browser downloads to complete installation

Would that be a change on keyman.com?

**Comment by mcdurdin** [comment]
> Would that be a change on keyman.com?

Yes. Low priority though, I'd prefer to review this alongside other improvements on macOS (e.g. the bundled keyboard+app install, new installer, etc.)
